uint8 id
bool in_cam
geometry_msgs/Pose pose
geometry_msgs/Pose gpose